approach/seaside-chats.qmd

```diff
@@ -55,6 +55,20 @@ No RSVP or preparation needed 😅 Bring snacks / mocktails / cocktails!
 We wanted to create a way for people to (re)connect informally with other Champions, Mentors, the Openscapes team, anyone in this Slack. I know many of you are keen to hear what others have been up to so come hang out! I'll facilitate so I promise it won't be awkward 😉 (This msg is pinned to this channel so you can find the details any time)
 :::
 
+## Land Acknowledgments
+
+We give a land acknowledgement during the welcome of every Cohort Call and Community Call. We recommend omitting the phrase "I would like to acknowledge" and jumping right to the point. Ideally we try to include an action so that people understand that these are more than just words. Here are some examples you can reuse or remix.
```

Comment:
```suggestion
We give a land acknowledgement during the welcome of every Cohort Call and Community Call. Following guidance from Ta7talíya Michelle Nahanee through her [Nahanee Creative](https://www.nahaneecreative.com/)
Territorial Acknowledgements Interactive Workshop, we omit the phrase "I would like to acknowledge" and jump right to the statements. Ideally we try to include an action so that people understand that these are more than just words. Here are some examples you can reuse or remix.
```